### PR TITLE
My changes - Add new creature Dog

### PR DESCRIPTION
**Pull request by alenazakh**
Add new creature Dog

### CODE REVIEW

File: src/main/java/org/zoo/Animals.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package org.example;
+package org.zoo;
 public abstract class Animals extends EarthCreatures {
```
Comment (Fulcrum81) [inline src/main/java/org/zoo/Animals.java]: Why is this class abstract? I don't see any abstract methods in it

File: src/main/java/org/zoo/EarthCreatures.java [hunk 2]
```diff
@@ -1,6 +1,4 @@
-package org.example;
-import java.util.ArrayList;
-import java.util.SortedMap;
+package org.zoo;
 
 public abstract class  EarthCreatures {
```
Comment (Fulcrum81) [inline src/main/java/org/zoo/EarthCreatures.java:3]: Why is this class abstract? I don't see any abstract methods in it

File: src/main/java/org/zoo/Animals.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 
     public Animals () {
```
Comment (Fulcrum81) [inline src/main/java/org/zoo/Animals.java]: Is the constructor necessary here with the category? I don't see it used anywhere

File: src/main/java/org/zoo/Fishes.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package org.example;
+package org.zoo;
 
 public class Fishes extends EarthCreatures {
```
Comment (Fulcrum81) [inline src/main/java/org/zoo/Fishes.java]: Is the constructor necessary here with the category? I don't see it used anywhere

File: src/main/java/org/zoo/Birds.java [hunk 5]
```diff
@@ -1,4 +1,4 @@
-package org.example;
+package org.zoo;
 public class Birds extends EarthCreatures {
 
     public Birds () {
```
Comment (Fulcrum81) [inline src/main/java/org/zoo/Birds.java]: Is the constructor necessary here with the category? I don't see it used anywhere

**Review by Fulcrum81 — COMMENTED** [review]
Overall I would prefer if the classes were somehow grouped in packages. Probably all animals in "animals" package, all birds in "birds" package etc. Would be easier to navigate.

File: src/main/java/org/zoo/Animals.java (hunk 1 above)
Comment (alenazakh) [reply src/main/java/org/zoo/Animals.java]: fixed

File: src/main/java/org/zoo/EarthCreatures.java (hunk 2 above)
Comment (alenazakh) [reply src/main/java/org/zoo/EarthCreatures.java:3]: make two methods abstract, leave as abstract

File: src/main/java/org/zoo/Animals.java (hunk 3 above)
Comment (alenazakh) [reply src/main/java/org/zoo/Animals.java]: constructor is removed

File: src/main/java/org/zoo/Fishes.java (hunk 4 above)
Comment (alenazakh) [reply src/main/java/org/zoo/Fishes.java]: constructor is removed

File: src/main/java/org/zoo/Birds.java (hunk 5 above)
Comment (alenazakh) [reply src/main/java/org/zoo/Birds.java]: constructor is removed

**Pull request merged**